gulpfile.js

```diff
@@ -0,0 +1,61 @@
+var gulp = require('gulp');
+var concat = require("gulp-concat");
+var uglify = require("gulp-uglify");
+var cleanCSS = require('gulp-clean-css');
+
+gulp.task('minify-all', ['vendor-styles', 'vendor-fonts', 'vendor-images', 'scripts']);
+
+gulp.task('vendor-styles', function () {
+    gulp.src([
+        './source/vendor/font-awesome/css/font-awesome.min.css',
+        './source/vendor/open-sans/styles.css',
+        './source/vendor/source-code-pro/styles.css',
+        './source/vendor/lightgallery/css/lightgallery.min.css',
+        './source/vendor/justified-gallery/justifiedGallery.min.css'
+        ])
+        .pipe(concat('vendor-styles.min.css'))
+        .pipe(cleanCSS())
+        .pipe(gulp.dest('./source/vendor/dest/css/'))
+});
+
+gulp.task('scripts', function () {
+    gulp.src([
+        './source/vendor/classie/classie.js',
+        './source/vendor/lightgallery/js/lightgallery.min.js',
+        './source/vendor/lightgallery/js/lg-thumbnail.min.js',
+        './source/vendor/lightgallery/js/lg-pager.min.js',
+        './source/vendor/lightgallery/js/lg-autoplay.min.js',
+        './source/vendor/lightgallery/js/lg-fullscreen.min.js',
+        './source/vendor/lightgallery/js/lg-zoom.min.js',
+        './source/vendor/lightgallery/js/lg-hash.min.js',
+        './source/vendor/lightgallery/js/lg-share.min.js',
+        './source/vendor/lightgallery/js/lg-video.min.js',
+        './source/vendor/justified-gallery/jquery.justifiedGallery.min.js',
+        './source/js/main.js',
+        './source/js/contact.js',
+        './source/js/form.js'
+    ])
+        .pipe(concat('vendor-scripts.min.js'))
```

Comment:
not a fan of the name as its not just vendor scripts id prefer just calling it main